ninja_ide/gui/ide.py

```diff
@@ -58,7 +58,7 @@
 from ninja_ide.gui.dialogs import about_ninja
 from ninja_ide.gui.dialogs import unsaved_files
 # from ninja_ide.gui.dialogs import schemes_manager
-# from ninja_ide.gui.dialogs import language_manager
+from ninja_ide.gui.dialogs import language_manager
```

Comment:
Could you write:
`from ninja_ide.gui.dialogs import language_manager, session_manager`
?